ext-src/packages/pypi/PyPiUtils.ts

```diff
@@ -17,11 +17,17 @@
 import { exec } from "../../utils/exec";
 import { PackageDependenciesHelper } from "../PackageDependenciesHelper";
 import { PyPIPackage } from './PyPIPackage';
+import * as os from 'os';
```

Comment:
can't you just `import { type } from 'os'`?

Reply:
> can't you just `import { type } from 'os'`?

Nevermind. I like the expression better as with the `os` object — it makes it a lot clearer.

Reply:
Done in: da1fc57

I think I was scared to do that because `type` is a reserved keyword in some languages? Did it and it looks fine to me?